src/shared/components/Banners/Banners.tsx

```diff
@@ -1,21 +1,32 @@
 import { Collapse } from '@mui/material';
 import { TransitionGroup } from 'react-transition-group';
+import { useCallback } from 'react';
```

Comment:
Could you please move the import from the react to the top, before other imports?